backend/db/seeds.rb

```diff
@@ -7,8 +7,11 @@
   User.delete_all
 
   5.times do
-    investor_account = FactoryBot.create(:account)
-    FactoryBot.create(:user, account: investor_account)
+    user = FactoryBot.create(:user)
+    investor_account = FactoryBot.create(:account, owner: user)
+    user.update!(account: investor_account, role: "investor")
+
+    investor_account.update!(owner: user)
```

Comment:
do we need this if the account was created with this owner?

Reply:
Right, we don't need this.